server/src/app.ts

```diff
@@ -31,14 +32,23 @@ app.use(removeJsonpCallback);
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.get('/', (_, res) => {
-  res.redirect('book');
+// set up rate limiter: maximum of ten requests per minute
+const limiter = rateLimit({
+  windowMs: 1 * 60 * 1000, // 1 minute
```

Comment:
Curious why this was added?

Reply:
There was security alert similar to this one - 
![image](https://user-images.githubusercontent.com/100614160/203188292-68695ba5-809c-4a73-9ac0-ccf7d0a6bbe4.png)
the linked material provided this as a solution.

Am open to suggestions if this needs to be removed

Reply:
I am not sure of any side effects that this change may introduce... I propose addressing it separately (if it indeed does appear again) to avoid introducing any potential issues with it in this PR.